test/functions_linearalgebra.jl

```diff
@@ -0,0 +1,67 @@
+using LinearAlgebra
+using NamedDims
+using NamedDims: names
+using Test
+
+# LinearAlgebra
+@testset "lu" begin
+    nda = NamedDimsArray{(:foo, :bar)}([1.0 2; 3 4])
+    x = lu(nda)
+    @test names(x.L) == (:foo, :_)
+    @test names(x.U) == (:_, :bar)
+    @test names(x.p) == (:foo,)
+    @test names(x.P) == (:foo, :foo)
+
+    # Idenity opperations should give back original names
+    @test names(x.P * nda) == (:foo, :bar)
+    @test names(x.L * x.U) == (:foo, :bar)
```

Comment:
I find the "Identity operations" comment a little misleading, since really `x.P * nda ≈ x.L * x.U` i.e `nda ≈ x.P * x.L * x.U` (becuase `I = S.P * S.P`)